### PR TITLE
Arch: new TwistedBridge object based on PathTwistedArray

### PR DESCRIPTION
**Pull request by vocx-fc**
It creates a "ribcage" twisting a solid profile around a path, then creates the "tunnel" by sweeping the internal wires of the repeated profiles along the path, and finally extrudes the "walkway" from a rectangle that follows the path.

The result is a "twisted bridge" object.

This pull request is a prototype, mostly to showcase what can be done with the "twisted array" object, introduced in #3841.

Forum thread: [PowerCopy in FreeCAD?](https://forum.freecadweb.org/viewtopic.php?f=23&t=49617)

---

- [x] Branch rebased on latest master `git pull --rebase upstream master`
- [x] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [x] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

### CODE REVIEW

File: src/Mod/Arch/ArchGui.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# *   License along with this program; if not, write to the Free Software   *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""Provide the graphical tools of the Arch workbench."""
+
+__title__ = "FreeCAD Arch GUI tools"
+__author__ = "Eliud Cabrera Castillo"
+__url__ = "http://www.freecadweb.org"
```
Comment (luzpaz) [inline src/Mod/Arch/ArchGui.py:25]: Do you mind using `https` instead?

**Comment by luzpaz** [comment]
@yorikvanhavre what do think about this PR? I'm concerned it will rot since @vocx-fc has departed the community. Maybe @marioalexis84 would be interested in adopting it ?

**Comment by yorikvanhavre** [comment]
I need to have a better look at it.. but yes it could be finished and merged

**Comment by carlopav** [comment]
If i can add my 2 cents, i'd prefer to drop the PR and turn It into a macro: the object it provides Is too specific, and could be modeled with other tools.

**Comment by yorikvanhavre** [comment]
I agree, plus it introduces a complex draft-like structure in Arch that I'd prefer to avoid :sweat_smile: I'll refactor this into a macro when I have a moment...

**Comment by berndhahnebach** [comment]
Following a link to the branch on the CI-repository:

https://gitlab.com/berndhahnebach/FreeCAD/-/commits/PR_5060

The CI-status is available on the latest commit of the branch.
If there is no status available the PR should be rebased on latest master.
Check pipeline branches to see if your PR has been run by the CI.

https://gitlab.com/berndhahnebach/FreeCAD/-/pipelines?scope=branches

**Comment by donovaly** [comment]
@yorikvanhavre  can this PR be made ready this week? If not, please give it the "for 0.21" label.

**Comment by donovaly** [comment]
So this PR will not be merged it seems and there will be another solution once the development of FC 0.21 will start.
Closing this PR. Once there is a solution for the issue, please create a new PR or reopen this one if necessary.

**Pull request closed without merging**